mozilla-release/browser/branding/cliqz/locales/browserconfig.properties

```diff
@@ -3,4 +3,4 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 # Do NOT localize or otherwise change these values
-browser.startup.homepage=about:home
+browser.startup.homepage=about:cliqz
```

Comment:
Not strictly necessary, as this url is overridden in `cliqz.cfg`.
Just for the sanity.